templates/kubernetes/terraform/environments/stage/application_iam_policy.tf

```diff
@@ -10,4 +10,15 @@ data "aws_iam_policy_document" "resource_access_backendservice" {
     resources = ["arn:aws:ec2:::stage-*"]
   }
   # can be more statements here
-}
\ No newline at end of file
+
+  statement {
```

Comment:
This service shouldn't need this access.

Reply:
done

Reply:
done